src/OpenSage.Launcher/OpenSage.Launcher.csproj

```diff
@@ -13,6 +13,10 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="System.CommandLine" Version="0.1.0-preview2-180220-2" />
```

Comment:
Looks like this is from MyGet, not NuGet? So you'll need to add the [dotnet-corefxlab MyGet feed](https://dotnet.myget.org/feed/dotnet-corefxlab/package/nuget/System.CommandLine) to [this file](https://github.com/OpenSAGE/OpenSAGE/blob/master/NuGet.config).

Reply:
Looks like I had it in my global Nuget config, so it built on my computer but failed in CI - I copied the line to NuGet.config and now it builds.